css/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+    font-family: 'montserrat-normal';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5388782)